### PR TITLE
chore(Travis): Add a PWA/deploying stage.

### PR DESCRIPTION
**Pull request by matanlurey**
PTAL @kevmoo, will merge internally if we like it.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -285,18 +304,23 @@ jobs:
...
+    # See https://docs.travis-ci.com/user/build-stages/#Build-Stages-and-Deployments.
+    - stage: deploying
+      script: skip
+      env: PKG="examples/hacker_news_pwa"
+      cache:
+        directories:
+          - examples/hacker_news_pwa/.dart_tool
+      before_deploy:
+        - pushd examples/hacker_news_pwa
+        - pub run build_runner build -o build --fail-on-severe
```
Comment (kevmoo) [inline .travis.yml]: `--config release`

Comment (kevmoo) [reply .travis.yml]: ...actually, now I *really* question the value of doing this...

Comment (matanlurey) [reply .travis.yml]: What do you mean?

Comment (kevmoo) [reply .travis.yml]: Wait. I get it now. Wow. Okay....

**Pull request closed without merging**